src/EPiServer.SocialAlloy.Web/Social/Tutorials/IContent Extensions/CommentExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿using EPiServer.Core;
+using EPiServer.ServiceLocation;
+using EPiServer.Social.Comments;
+using EPiServer.Social.Comments.Core;
+using EPiServer.Social.Common;
+using System;
+using System.Collections.Generic;
+
+namespace epiAlloySite
+{
+    /// <summary>
+    /// This class implements extension methods on an instance of Episerver IContent to support
+    /// adding and retrieving Episerver Social comments posted on that instance.
+    /// </summary>
+    public static class CommentExtensions
+    {
+        // Format specifiers for building target content and author strings used to form 
+        // references in Episerver Social. 
+        private const string userReferenceFormat = "user://{0}";
+        private const string resourceReferenceFormat = "resource://{0}";
+        
+        /// <summary>
+        /// Returns the Episerver Social Comment service used for managing comments.
+        /// </summary>
+        private static ICommentService CommentService
+        {
+            get
+            {
+                return ServiceLocator.Current.GetInstance<ICommentService>();
```

Comment:
@cbanner added static ctor, and updated the blog document too. 
Thanks for reviewing.